src/scripts/deploy_ecs_scheduled_task.sh

```diff
@@ -8,6 +8,11 @@ fi
 
 ORB_STR_RULE_NAME="$(circleci env subst "${ORB_STR_RULE_NAME}")"
 
+if [[ -n $ORB_STR_RULE_NAME ]]; then
+    echo "To deploy with an shceduled task, you must provide a rule name"
```

Comment:
There is a typo here: scheduled
This makes the `rule_name` parameter mandatory right?